packages/docusaurus-theme-classic/src/theme/Layout/styles.css

```diff
@@ -10,6 +10,7 @@
 body {
   margin: 0;
   padding-top: var(--ifm-navbar-height);
+  transition: .4s ease color;
```

Comment:
@yangshun here using a transition effect on body for theme toggling, would like your opinion on this :)

Reply:
Use `var(--ifm-transition-fast)`